engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -1105,6 +1105,15 @@ public void handle(Signal sig)
         org.apache.log4j.Logger.getRootLogger(),
         org.apache.log4j.Logger.getLogger(ApexCli.class)
     }) {
+
+     /*
+      * As the default logger level is INFO. So all the appenders of this logger are
+      * unable to log to console (in case of ConsoleAppender) as parent or ancestor
+      * is INFO level. So overiding it to DEBUG level and threshold level for Console-
+      * appender is set through command line arguments.
+      */
+      logger.setLevel(Level.DEBUG);
```

Comment:
Can you set it to the verbose level, instead of setting it to debug for all cases?